### DISCUSSION TITLE
2022/05/20/webpack-aliases-with-sanity/

### DISCUSSION THREAD

**Discussion by giscus** [KittyGiraudel/site#159] [Announcements]
# 2022/05/20/webpack-aliases-with-sanity/

I’m Kitty Giraudel, a non-binary trans frontend developer based in Berlin, focused on accessibility and inclusivity.

https://kittygiraudel.com/2022/05/20/webpack-aliases-with-sanity/

**Comment by Jamiewarb** [comment]
Thanks for this.

Sanity's default tsconfig complaints about version 3. That can be remedied with the below. Not as neat, but solves the error easily enough

```ts
import path from 'path';
import { defineCliConfig } from 'sanity/cli';

export default defineCliConfig({
  api: {},

  vite: config => {
    if (!config.resolve) config.resolve = {};

    config.resolve.alias = {
      ...config.resolve.alias ?? {},
      '@': path.resolve(__dirname, '..', 'src'),
    }

    return config;
  },
})
